spec/rails_helper.rb

```diff
@@ -33,7 +33,7 @@
 end
 RSpec.configure do |config|
   # Remove this line if you're not using ActiveRecord or ActiveRecord fixtures
-  config.fixture_path = "#{::Rails.root}/spec/fixtures"
+  # config.fixture_path = "#{::Rails.root}/spec/fixtures"
```

Comment:
should this be here?

Reply:
yes, I just commented it out like other lines in the file because we are not using rspec fixtures